.github/workflows/validate-integration-files.yml

```diff
@@ -0,0 +1,71 @@
+name: Validate integration files
+
+on:
+  pull_request:
+    paths:
+      - 'integrations/**'
+
+jobs:
+  validate-files:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Setup
+      run: |
+        pip install toml-cli yq
+
+    - name: Check Ocean version 🌊
+      run: |
+        changed_dirs=$(git diff --name-only origin/main origin/${{ github.head_ref }} | grep 'integrations/' | cut -d'/' -f 1,2 | sort -u)
+        package_version=$(curl -s https://pypi.org/pypi/port-ocean/json | jq -r '.info.version')
+        for dir in $changed_dirs; do
+          pyproject_file=$(find $dir -name 'pyproject.toml' -not -path "**/.venv/*")
+          if [ -n "$pyproject_file" ]; then
+            installed_version=$(toml get  tool.poetry.dependencies.port_ocean.version --toml-path $pyproject_file)
+            is_version_updated=$(python -c "from packaging import version;print(version.parse('$installed_version'.lstrip('^')) >= version.parse('$package_version'))")
```

Comment:
This might be a bit risky because we are relying on the pre-installed python in the github runner, while I don't expect it to change into an incompatible version very soon, it still might happen, is there a way we can achieve this is bash?

If not, we can also just use the `setup-python` action, it will add a few more seconds to the CI but it will be more reproducible (we also don't need any packages so it shouldn't be too bad)